Sources/MapboxDirections/RouteRefreshResponse.swift

```diff
@@ -157,4 +247,25 @@ extension Route {
             }
         }
     }
+    
+    /**
+     Merges the closures of the given route’s legs into the receiver’s legs.
+
+     - parameter refreshedRoute: The route containing leg closures to merge into the receiver. If this route contains fewer legs than the receiver, this method skips legs from the beginning of the route to make up the difference, so that merging the closures from a one-leg route affects only the last leg of the receiver.
+     - parameter legIndex: The index of a leg, from which to start applying the refreshed closures.
+     - parameter legShapeIndex: Index of a geometry of the `legIndex` leg, where to start refreshing from.
+     */
+    public func refreshLegClosures(from refreshedRoute: RouteRefreshSource, legIndex: Int = 0, legShapeIndex: Int = 0) {
```

Comment:
`refreshLegClosures` method looks just like `refreshLegIncidents`, but contains a specific implementation with 1 parameter.  

Maybe it's better to use the same approach for all three methods here, either default parameters values or separate method. Using different solutions for the same situation might be confusing